File: src/main/resources/static/js/shuttle-activity.js
function ShuttleActivity() {
  var self = this;

  self.shuttleId = 0;
  self.driverId = 0;
  self.assignmentId = 0;
  self.latitude = 0.0;
  self.longitude = 0.0;
  self.currentStopIndex = 0;
  self.heading = 0.0;
  self.status = 'NONE';
  self.latitude = null;
  self.longitude = null;

  return self;
}

File: src/main/kotlin/com/polaris/app/driver/service/bo/ShuttleActivity.kt
package com.polaris.app.driver.service.bo

import com.polaris.app.dispatch.controller.adapter.enums.ShuttleState
import java.math.BigDecimal

data class ShuttleActivity(
        val shuttleID: Int,
        val driverID: Int,
        val assignmentID: Int,
        val assignmentStopID: Int,
        val index: Int,
        val latitude: BigDecimal,
        val longitude: BigDecimal,
        val heading: BigDecimal,
        val status: ShuttleState
)